src/main.cpp

```diff
@@ -181,6 +181,17 @@ bool vcMain_TakeScreenshot(vcState *pProgramState)
   if (currSize.x - pProgramState->settings.screenshot.resolution.x > 32 || currSize.y - pProgramState->settings.screenshot.resolution.y > 32)
     return true;
 
+  udFindDir* poutputPath(nullptr);
+  udResult result = udOpenDir(&poutputPath, pProgramState->settings.screenshot.outputPath);
+  if (result != udR_Success)
+  {
+    //Folder may not exist, try to create.
+    result = udCreateDir(pProgramState->settings.screenshot.outputPath);
```

Comment:
This function is not recursive, if a user were to manually type in a path that contained multiple folders that don't exist this would fail. However, there is a PR pending in udCore for this functionality.

Reply:
I understand. What would you suggest? Wait for this function to come in and redo this item?

Reply:
I think it's fine to merge as-is - @peterAdams1014 can you get the udCore work prioritised? As far as I'm aware, there's a couple issues with the unit tests, but otherwise it's good to go.